eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -10,7 +10,7 @@
       "windows-2019": { "OSVmImage": "MMS2019", "Pool": "azsdk-pool-mms-win-2019-general" },
       "macOS-10.15": { "OSVmImage": "macOS-10.15", "Pool": "Azure Pipelines" }
     },
-    "PythonVersion": [ "pypy3.7", "3.7", "3.8" ],
+    "PythonVersion": [ "pypy3.7", "3.7", "3.8", "3.9"],
```

Comment:
Just a heads up, the reason that `3.9` is excluded in this set` is because it's actually being run with coverage enabled  from the `include` section below on line 23.

Reply:
Thank you so much for the detailed explanation! I excluded 3.9 from the cryptography-dependency set as well!